### ISSUE TITLE
Por hacer

### ISSUE THREAD

**Issue by mlozanoqf** [carolinavcha/tesis#1]
Veo a @carolinavcha pero no veo al resto del equipo como _contributors_. Sugiero que el resto se integre a Github para que ustedes colaboren y que yo pueda supervisar su progreso con mayor detalle. No he revisado su avance pero sugiero que el rmd haga un pdf en lugar de un html. Este mismo documento se puede usar para el resto de las secciones del trabajo, con lo cual espero ver también aquí avances sobre la revisión literaria por ejemplo. Cuando tengan un avance en este issue me dicen para revisarlo.

**Comment by mlozanoqf** [comment]
Ya veo a todos los miembros del equipo como colaboradores. Ahora solo falta un avance más claro. Saludos.

**Comment by mlozanoqf** [comment]
Tengo un repositorio llamado CC-diss-rmd que adapté de otro para que sea el template para su tesis. El archivo main es el que llama al resto de secciones (child). También incluye un documento bib para las referencias o bibliografía. Hay que usar ese template lo más pronto posible para que se familiaricen con él. Saludos.

**Comment by carolinavcha** [comment]
Listo profe, ya se agregó el template al .rmd en el que estamos trabajando.

**Comment by mlozanoqf** [comment]
Not quite. En realidad el main casi no debe tener contenido. El contenido está en los otros documentos que llama el main, que se le llaman "child". Cuando haces knit del main, el main llama al resto y produce el pdf completo. Lo que agregaste creo que debe estar dentro del rmd de resultados.

**Comment by carolinavcha** [comment]
ok entiendo. lo resuelvo enseguida.
